File: Client/src/app/features/asset-movements/data-access/models/create-asset-movement-response.ts
import { AssetMovementType } from "./asset-movement-type";

export interface CreateAssetMovementResponse {
  readonly id: string;
  readonly assetId: string,
  readonly storeId: string,
  readonly serialNumber: string;
  readonly employeeNumber: string;
  readonly type: AssetMovementType;
  readonly createdAtUtc: string;
}

File: AssetManagement.Application/AssetMovements/CreateAssetMovement/CreateAssetMovementResponse.cs
using AssetManagement.Domain.Entities.AssetMovements;

namespace AssetManagement.Application.AssetMovements.CreateAssetMovement
{
    public sealed record CreateAssetMovementResponse
    (
        Guid Id,
        Guid AssetId,
        Guid StoreId,
        string SerialNumber,
        string EmployeeNumber,
        AssetMovementType Type,
        DateTime CreatedAtUtc
        );
}
